src/auth/externalAccountAuthorizedUserClient.ts

```diff
@@ -39,6 +39,7 @@ import {
  */
 export const EXTERNAL_ACCOUNT_AUTHORIZED_USER_TYPE =
   'external_account_authorized_user';
+const DEFAULT_TOKEN_URL = 'https://sts.googleapis.com/v1/token';
```

Comment:
same here, needs to be derived w/ universe_domain

lets also add a test for this (everywhere where we've added a default for suppliers)